src/components/ImageDepViewer.js

```diff
@@ -56,12 +56,16 @@ export const ImageDepViewer = ({ packageLink }) => {
           ])
         ]),
         tbody(
-          _.map(({ name, version }) => {
-            return tr({ key: name }, [
-              td({ style: { paddingRight: '1rem' } }, [name]),
-              td([version])
-            ])
-          }, _.filter({ tool: selectedTool }, packages))
+          _.flow(
+            _.filter({ tool: selectedTool }),
+            _.sortBy(({ name }) => _.toLower(name)),
```

Comment:
Have we gotten the UX team's approval on sorting such that packages with uppercase letters are interleaved with packages with lowercase letters? We don't seem to be doing that in other places in the codebase so wanted to double-check. I don't feel strongly either way but it'd be nice to be consistent :)

Reply:
I had not asked them. Just posted about it in Slack though.

Reply:
Approved: https://broadinstitute.slack.com/archives/C34GAKQ5R/p1645111855628929?thread_ts=1645046899.125539&cid=C34GAKQ5R